app/views/katello/api/v2/subscriptions/show.json.rabl

```diff
@@ -59,10 +59,3 @@ node :activation_keys do |subscription|
     }
   end
 end
-
-node :host, :if => lambda { |sub| sub && sub.host } do |subscription|
```

Comment:
@jlsherrill :smile: I know, I couldn't think of a way to remove it when I refactored subscriptions, maybe we can brainstorm more on it later.